docs/source/operations/getting-started.rst

```diff
@@ -0,0 +1,354 @@
+***************
+Getting started
+***************
+
+
+This page gives you direction how to run locally the substra stack. This deployment is made of:
+
+* 1 orchestrator (running in standalone mode, runnign its own local DB)
+* 2 backends (running in two organisations, ``org-1`` and ``org-2``)
+* 1 frontend
+
+It allows you to run the examples and start using substra SDK (also known as substra).
+
+Prerequisites
+=============
+
+Hardware
+--------
+
+The following table indicates the resources needed to run the Substra stack locally. The minimal profile represents resources needed to be able to run the stack, whereas the recommended profile describes how much is needed to have a comfortable development experience.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * -
+     - CPU
+     - Hard drive space (Go)
+     - RAM
+   * - Minimal
+     - 2
+     - 35
+     - 4
+   * - Recommended
+     - 4-8
+     - 50
+     - 8
+
+.. caution::
+   Choose wisely the parameters passed to Kubernetes as it might try to use all the allocated resources without regards for your system.
+
+Software
+--------
+
+* `git <https://git-scm.com/downloads>`_
+* `Docker <https://docs.docker.com/>`_ (>= 4.0.0)
+*  sed
+
+   .. caution::
+      On MacOS you need `gsed`.
+
+* k3d/k3s (>= 5.0.0)
+* `kubectl <https://kubernetes.io/>`_
+* `skaffold <https://skaffold.dev/>`_
+* `helm 3 <https://helm.sh/>`_ (>= 3.7.0)
+*  `nodeJS <https://nodejs.org/>`_ (== 16.13.0)
+
+Instructions for Mac
+^^^^^^^^^^^^^^^^^^^^
+
+First, install `Homebrew <https://brew.sh/>`_, then run the following commands:
+
+.. code-block:: bash
+
+   brew install k3d
+   brew install kubectl
+   brew install skaffold
+   brew install helm
+   brew install gsed # Needed for k3s-create.sh
+
+
+First time configuration
+========================
+
+1. Execute the script :download:`k3-create.sh<./getting-started/k3-create.sh>`. This script deletes the existing cluster, recreates a new one and apply a patch for SSL.
+
+   1. Download :download:`k3-create.sh<./getting-started/k3-create.sh>`.
+   2. Make the script executable.
+
+      .. code-block:: bash
+
+         chmod +x ./k3-create.sh
+
+   1. Run the script
+
+      .. code-block:: bash
+
+         ./k3-create.sh
+
+   .. tip::
+      This script can be used to reset your development environment.
+
+2. Add the following line to ``/etc/hosts`` to allow the communication between your local cluster and the host (your machine):
+
+   .. code-block:: text
+
+      127.0.0.1 orchestrator.org-1.com orchestrator.org-2.com substra-frontend.org-1.com substra-frontend.org-2.com substra-backend.org-1.com substra-backend.org-2.com
+
+3. Add the helm repositories
```

Comment:
```suggestion
3. Add the Helm repositories
```

Reply:
So should I capitalize all references to Helm? If so, should I apply it to other softwares (e.g. Docker, Skaffold)

Reply:
If the official name is capitalized, I am in favor of having it capitalized in the doc.

Reply:
Fair enough, Docker also mentions that the name has to be capitalised.